moyo/src/lib.rs

```diff
@@ -85,6 +85,31 @@ use crate::symmetrize::{orbits_in_cell, StandardizedCell, StandardizedMagneticCe
 
 use nalgebra::Matrix3;
 
+/// The crystal system of a structure.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum CrystalSystem {
```

Comment:
We already have CrystalSystem class in `data/classification.rs`.
https://github.com/spglib/moyo/blob/4748c80715fcb9d2c1a75ca6615b35648beb8f60/moyo/src/data/classification.rs#L103-L111

I am sorry that you already implemented this, but can I take over this issue? I am thinking to add a python function `get_space_group_type` to return crystal systems and other classifications for space group types. 
https://github.com/spglib/moyo/issues/52

Reply:
ofc, feel free to take over/make any changes

Reply:
if you make it a python function, will the same information also be available on the rust side?

Reply:
The Rust side already has this information: space group type -> arithmetic crystal class -> geometric crystal class -> crystal system. So, I think the accessor for the Python interface is enough for implementation.

Reply:
ah, i should have read the code more carefully. at least i understand it better now